src/CalcManager/CalculatorHistory.cpp

```diff
@@ -41,7 +41,8 @@ unsigned int CalculatorHistory::AddItem(_In_ shared_ptr<HISTORYITEM> const &spHi
     }
 
     m_historyItems.push_back(spHistoryItem);
-unsigned int lastIndex = static_cast<unsigned>(m_historyItems.size() - 1);
+    unsigned int lastIndex = static_cast<unsigned>(m_historyItems.size() - 1);
+    return lastIndex;
 return lastIndex;
```

Comment:
As @janisozaur pointed out, the `suggestion` feature at the moment introduced _two_ return statements.
For now, do you think you can just fix this manually?

Reply:
Sure